website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -124,6 +124,12 @@ Envoy requires an initial bootstrap configuration file. You can either create th
 
 Connect to a local Consul client agent and run the [`consul connect envoy` command](/consul/commands/connect/envoy) to create the Envoy bootstrap configuration. The command either outputs the bootstrap configuration directly to stdout or generates the configuration and issues an `exec` command to the Envoy binary as a convenience wrapper. For more information about using `exec` to bootstrap Envoy, refer to [Exec Security Details](/consul/commands/connect/envoy#exec-security-details).
 
+If you experience issues when bootstrapping Envoy proxies from the CLI, use the
+`-enable-config-gen-logging` flag to enable debug message logging. These logs can
+help you troubleshoot issues during Envoy bootstrapping. For more information about
+available flags and parameters, refer to the
+[`consul connect envoy CLI` reference](/commands/connect/envoy).
```

Comment:
```suggestion
[`consul connect envoy CLI` reference](/consul/commands/connect/envoy).
```
Hi 👋🏻 Now that the [link migration](https://github.com/hashicorp/consul/pull/15976) has been released, I'm going through open PRs to resolve any issues. This is the new format that should be used for inline docs links. Let me know if you have questions!